main/lib/src/RunControl.cc

```diff
@@ -80,8 +82,12 @@ namespace eudaq {
     SendCommand("CONFIG", to_string(config));
     if (config.SetSection("RunControl")) {
       m_runsizelimit = config.Get("RunSizeLimit", 0LL);
+      m_runeventlimit = config.Get("RunEventLimit", 0LL);
+      m_nextconfigonfilelimit = config.Get("NextConfigFileOnFileLimit", false);
```

Comment:
I don't completely get what this feature is doing. Is it correct that you want to have the RunControl automatically reconfigure with another config file after stopping a run via the RunSizeLimit and restart a new one after the reconfiguration?

Quite a useful feature I guess, especially when thinking about overnight runs scanning over a set of parameters.

However, I would suggest renaming the parameter a bit to make it more clear what it's supposed to do, maybe `NextConfigFileOnRunChange`? Also, how do I specify what the range for config files is?